package.json

```diff
@@ -30,13 +30,14 @@
   "author": "Kent C. Dodds <me@kentcdodds.com> (https://kentcdodds.com)",
   "license": "MIT",
   "dependencies": {
-    "@babel/runtime": "^7.9.2",
-    "babel-plugin-macros": "^2.8.0",
+    "@babel/runtime": "^7.12.5",
+    "babel-plugin-macros": "^3.0.1",
     "require-from-string": "^2.0.2"
   },
   "devDependencies": {
+    "@types/babel__core": "^7.1.12",
     "ast-pretty-print": "^2.0.1",
-    "babel-plugin-tester": "^8.0.1",
+    "babel-plugin-tester": "^10.0.0",
```

Comment:
is bumping stuff in-scope? Not sure it's out prerogative to decide that

Reply:
The charter is "as per [`babel-plugin-codegen`](https://github.com/kentcdodds/babel-plugin-codegen/pull/34/files?file-filters%5B%5D=.json) so in this case it's fine IMO.

Generally there is some scope to bump versions if it improves the TS compatibility as well, or in this case, `kcd-scripts` did not support TS at the version in this repo.  Most maintainers aren't going to complain if you're updating to a _newer_ version of something either (one less job for them to do).